lms/templates/instructor/instructor_dashboard_2/membership.html

```diff
@@ -12,15 +12,24 @@
         ${_("You will not get notification for emails that bounce, so please double-check spelling.")}
         <textarea rows="6" name="student-ids" placeholder="${_("Email Addresses/Usernames")}" spellcheck="false"></textarea>
     </label>
-    <input type="hidden" id="is_course_white_label" value="${section_data['is_white_label']}">
-    % if section_data['is_white_label']:
+
+    <div class="role">
         <label>
-            ${_("Enter the reason why the students are to be manually enrolled or unenrolled.")}
-            ${_("This cannot be left blank and will be recorded and presented in Enrollment Reports.")}
-            ${_("Therefore, please give enough detail to account for this action.")}
-            <textarea rows="2" id="reason-field-id" name="reason-field" placeholder="${_('Reason')}" spellcheck="false"></textarea>
+            ${_("Role of the people being enrolled.")}
+            <select name="role">
+                % for role in section_data['enrollment_role_choices']:
+                  <option value="${role}">${role}</option>
+                % endfor
+            </select>
```

Comment:
Is there a check somewhere in the UI that Role has to be selected during an enrollment?
Also, if the action is unenroll then wondering if we need to make Role required?

Reply:
@bill-filler I can add a check, but if we don't have an empty select option in the dropdown, then one of the desired options will be chosen automatically.

Reply:
I think leaving as is will be ok